ipaserver/install/dsinstance.py

```diff
@@ -615,14 +615,19 @@ def __add_default_schemas(self):
             # Does not apply with newer DS releases
             pass
 
+    def start(self, *args, **kwargs):
+        super(DsInstance, self).start(*args, **kwargs)
+        api.Backend.ldap2.connect(size_limit=LDAPClient.size_limit,
+                                  time_limit=LDAPClient.time_limit)
```

Comment:
That's a temporary change that is refactored in a later commit, the final code calls `api.Backend.ldap2.connect()`.